openquakeplatform/setup.py

```diff
@@ -74,16 +74,16 @@ def read(*rnames):
         "django-activity-stream==0.4.5beta1",
         "django-announcements==1.0.2",
         "django-appconf==0.6",
-        "django-chained-multi-checkboxes==0.4",
-        "django-chained-selectbox==0.2.1",
+        "django-chained-multi-checkboxes >= 0.4, < 0.5",
```

Comment:
In this way, with an old installation with the old tagged 0.4 `django-chained-multi-checkboxes` will not be updated to the new version

```
Requirement already satisfied (use --upgrade to upgrade): django-chained-multi-checkboxes>=0.4,<0.5 in /home/ubuntu/platform-env/lib/python2.7/site-packages (from openquakeplatform==1.3.0-gitba6ef10)
```